test/SILGen/`

```diff
@@ -0,0 +1,17 @@
+// RUN: %target-swift-emit-silgen -parse-as-library %s
```

Comment:
This file is called (`` ` ``), is this expected?

Reply:
No, that's a mistake. oops